### PR TITLE
Updated Indexer from the Shooter branch (no more ToF sensor and new methods)

### PR DESCRIPTION
**Pull request by spaceghost24**

### CODE REVIEW

**Review by TomP6 — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 1]
```diff
@@ -67,6 +32,22 @@ public double getWheelVelocity() {
     return 0.0;
   }
 
+  public void isCurrentCargoColored(){
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/Indexer.java]: change to isCurrentCargoRed (apparently FMS tells us if our alliance color is red or not)

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 2]
```diff
@@ -67,6 +32,22 @@ public double getWheelVelocity() {
...
+
+  }
+
+  public void isNextCargoColored(){
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/Indexer.java]: change to isNextCargoRed (apparently FMS tells us if our alliance color is red or not)

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 3]
```diff
@@ -67,6 +32,22 @@ public double getWheelVelocity() {
...
+
+  }
+
+  public void getNumCargo(){
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/Indexer.java]: return int

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 4]
```diff
@@ -67,6 +32,22 @@ public double getWheelVelocity() {
...
+
+  }
+
+  public void getCargoPositions(){
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/Indexer.java]: 2 methods?
return booleans?
getIsCurrentCargoPresent()
getIsNextCargoPresent()

**Review by r-wallis — CHANGES_REQUESTED** [review]

**Review by BenWertz — CHANGES_REQUESTED** [review]
The new methods need to have the right return types: if the return type is 'void' they can't return anything. For `getNumCargo()` it should return an int, but I'm not sure what `getCargoPositions()`, `isCurrentCargoColored`, or `isNextCargoColored` should return. Maybe for the cargo color, instead of returning the color we return a boolean indicating whether or not it's red.

**Review by r-wallis — APPROVED** [review]

**Review by BenWertz — APPROVED** [review]

**Pull request merged**